src/loaders/express.js

```diff
@@ -35,6 +42,29 @@ module.exports = (app) => {
     app.use(passport.initialize());
     app.use(passport.session());
 
+    const mongo_uri = config.db_uri;
+
+    // Set the API call rate limit only on production
+    if (process.env.NODE_ENV === "production") {
```

Comment:
We need to make sure that the servers in production are actually launched with `NODE_ENV=production` because of this. Commenting here so that I don't forget to quickly check this.

Reply:
`niployments` runs `Dockerfile-prod`. `nijobs-be`'s `Dockerfile-prod` runs `npm run prod`. However, `npm run prod` does not set `NODE_ENV` to `production`.

So, if you don't mind, this needs to be set in package.json for this PR to be effective :slightly_smiling_face:

Reply:
(Furthermore, we should really update the node versions in the dockerfiles, as they are being used for production, CI and etc. Creating an issue to track that now)

Reply:
Resolving as this was tackled